benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -211,8 +212,11 @@ async def send_request(
   async with aiohttp.ClientSession(timeout=timeout,trust_env=True) as session:
     while True:
       try:
+        active_requests_metric.inc()
```

Comment:
Shouldn't we count this after the request is posted? Will it be accurate if we do it just before sending the request?

Reply:
Does aiohttp client session have some maintained prometheus exporter package that makes sense to use in our scenario?

Reply:
From what I gather it doesn't, looks like theres a few packages that do this
- https://pypi.org/project/prometheus-async/
- https://pypi.org/project/aiohttp-prometheus-client/

Reply:
ack, that is fine.

One more thought, is client session able to have custom middleware that might simplify this?

We probably will also need this supported with streaming as well.

Reply:
Added